cargo/spec/dependabot/cargo/update_checker/requirements_updater_spec.rb

```diff
@@ -151,17 +151,34 @@
         end
 
         context "when there were multiple range specifications" do
-          let(:req_string) { "> 1.0.0, < 1.2.0" }
-          its([:requirement]) { is_expected.to eq("> 1.0.0, < 1.6.0") }
+          context "with `less then`" do
+            let(:req_string) { "> 1.0.0, < 1.2.0" }
+            its([:requirement]) { is_expected.to eq("> 1.0.0, < 1.6.0") }
 
-          context "when already valid" do
-            let(:req_string) { "> 1.0.0, < 1.7.0" }
-            its([:requirement]) { is_expected.to eq(req_string) }
+            context "when already valid" do
+              let(:req_string) { "> 1.0.0, < 1.7.0" }
+              its([:requirement]) { is_expected.to eq(req_string) }
+            end
+
+            context "when including a pre-release" do
+              let(:req_string) { ">=1.2.0, <1.4.0-dev" }
+              its([:requirement]) { is_expected.to eq(">=1.2.0, <1.6.0") }
+            end
           end
 
-          context "when including a pre-release" do
-            let(:req_string) { ">=1.2.0, <1.4.0-dev" }
-            its([:requirement]) { is_expected.to eq(">=1.2.0, <1.6.0") }
+          context "with `less then equal`" do
```

Comment:
Non-blocking/nitpick: technically `less than or equal to`

Reply:
Fixed